### PR TITLE
Translational kinematics

### PR DESCRIPTION
**Pull request by moorepants**

### CODE REVIEW

**Comment by moorepants** [comment]
@jtheinen this is ready for review too. It'd a bit rougher, but I can polish before Friday. It's complete for review purposes.

**Comment by moorepants** [comment]
Also, in this one, ignore the files from #20 because those should be reviewed in that PR.

File: translational.rst [hunk 1]
```diff
@@ -0,0 +1,547 @@
...
+
+where point :math:`T` is a point that conicides with :math:`R` at that instant.
+
+Combined with the two point theorem for :math:`T`, you can write:
+
+.. math::
+
+   {}^N\bar{v}^R = {}^B\bar{v}^R + {}^N\bar{v}^S + {}^N\bar{\omega}^B \times \bar{r}^{R/S}
+
+In our kinetic sculpture example, ff the pigeon :math:`R` is walking at a
```
Comment (jtheinen) [inline translational.rst:319]: ff = if

**Review by jtheinen — COMMENTED** [review]
The chapter translational kinematics is really understandable, coherent and well explained!

**Comment by jtheinen** [comment]
I was thinking, maybe it would be more understandable to start with translational kinematics then do angular kinematics. It is easier to comprehend the translational chapter and you could even use the same example of the kinetic structure in rotterdam for the angular velocity, body fixed rotations and addition of angular acceleration

**Comment by moorepants** [comment]
> I was thinking, maybe it would be more understandable to start with translational kinematics then do angular kinematics. It is easier to comprehend the translational chapter and you could even use the same example of the kinetic structure in rotterdam for the angular velocity, body fixed rotations and addition of angular acceleration

I think I need to keep them in that order, but I could introduce the example in the angular chapter and use it in both. Good idea.

**Comment by moorepants** [comment]
@jtheinen you only found a single error in this page?

**Pull request merged**